src/burn/drv/pre90s/d_galaga.cpp

```diff
@@ -2088,21 +2088,8 @@ static struct N54XX_Sample_Info_Def galagaN54xxSampleList[] =
 
 static struct Machine_Config_Def galagaMachineConfig =
 {
-	.cpus                   = galagaCPU,
-	.wrAddrList             = galagaWriteTable,
-	.rdAddrList             = galagaReadTable,
-	.memMapTable            = galagaMemTable,
-	.sizeOfMemMapTable      = GALAGA_MEM_TBL_SIZE,
-	.romLayoutTable         = galagaROMTable,
-	.sizeOfRomLayoutTable   = GALAGA_ROM_TBL_SIZE,
-	.tempRomSize            = 0x2000,
-	.tilemapsConfig         = galagaTilemapConfig,
-	.drawLayerTable         = galagaDrawFuncs,
-	.drawTableSize          = GALAGA_DRAW_TBL_SIZE,
-	.getSpriteParams        = galagaGetSpriteParams,
-	.reset                  = galagaReset,
-	.customRWTable          = galagaCustomICRW,
-	.n54xxSampleList        = galagaN54xxSampleList
+	galagaCPU,galagaWriteTable,galagaReadTable,galagaMemTable,GALAGA_MEM_TBL_SIZE,galagaROMTable, GALAGA_ROM_TBL_SIZE,0x2000,
+	galagaTilemapConfig, galagaDrawFuncs, GALAGA_DRAW_TBL_SIZE, galagaGetSpriteParams,galagaReset,galagaCustomICRW,galagaN54xxSampleList
 };

```

Comment:
Can you make this look better at least?
f.ex:
{
  /*cpus*/ *tab* galagaCPU,
  /*wrAddrList*/ *tab* galagaWriteTable,

I can do it if you don't want to :)

Reply:
I sure can :)

Reply:
Ah, saw you did it already :)

Reply:
oops, can ya please remove d_galaga from your PR, then I can merge it

Reply:
should be ok now